From 3294a6e1a7d7a6bea1cb02ec671aaa0d350b57f5 Mon Sep 17 00:00:00 2001
From: Jasper St. Pierre <jstpierre@mecheye.net>
Date: Fri, 2 Sep 2011 16:01:24 -0400
Subject: [PATCH 1/2] theme: Lighten up the sent message color, clean up

Properly apply "received" style and drop unused border-radius styles
now that the messages have no background color

https://bugzilla.gnome.org/show_bug.cgi?id=658096
---
 data/theme/gnome-shell.css |    7 ++-----
 1 files changed, 2 insertions(+), 5 deletions(-)

diff --git a/data/theme/gnome-shell.css b/data/theme/gnome-shell.css
index 4487159..0e09866 100644
--- a/data/theme/gnome-shell.css
+++ b/data/theme/gnome-shell.css
@@ -1329,9 +1329,8 @@ StTooltip StLabel {
     padding: 8px 0;
 }
 
-.chat-sent {
+.chat-received {
     padding-left: 4px;
-    border-radius: 4px;
 }
 
 .chat-received:rtl {
@@ -1341,8 +1340,7 @@ StTooltip StLabel {
 
 .chat-sent {
     padding-left: 18pt;
-    border-radius: 4px;
-    color: #7E7E7E;
+    color: #959595;
 }
 
 .chat-sent:rtl {
@@ -1352,7 +1350,6 @@ StTooltip StLabel {
 
 .chat-meta-message {
     padding-left: 4px;
-    border-radius: 4px;
     font-size: 9pt;
     color: #bbbbbb;
 }
-- 
1.7.3.4

